packages/core/src/browser/style/tabs.css

```diff
@@ -410,3 +410,12 @@
   flex-flow: row nowrap;
   min-width: 100%;
 }
+
+.p-TabBar-tab .theia-tab-icon-label {
+  flex: 1;
+}
+
+.p-TabBar-tab.p-mod-closable:hover .theia-tab-icon-label,
+.p-TabBar-tab.p-mod-current .theia-tab-icon-label {
+  overflow: hidden;
+}
```

Comment:
It appears that this styling causes the following regression, and possibly others:


https://user-images.githubusercontent.com/40359487/231267346-cbf4ed52-8220-4a3b-8d09-16937e444924.mov